lib/content-services/src/lib/category/services/category.service.ts

```diff
@@ -58,14 +58,14 @@ export class CategoryService {
     }
 
     /**
-     * Creates subcategory under category with provided categoryId
+     * Creates subcategories under category with provided categoryId
      *
      * @param parentCategoryId The identifier of a parent category.
-     * @param payload Created category body
-     * @return Observable<CategoryEntry>
+     * @param payload List of categories to be created.
+     * @return Observable<CategoryPaging | CategoryEntry>
      */
-    createSubcategory(parentCategoryId: string, payload: CategoryBody): Observable<CategoryEntry> {
-        return from(this.categoriesApi.createSubcategory(parentCategoryId, [payload], {}));
+    createSubcategories(parentCategoryId: string, payload: CategoryBody[]): Observable<CategoryPaging | CategoryEntry> {
```

Comment:
Something is wrong with type I think - you say CategoryPaging | CategoryEntry but if you check what that function does it just returns following line: 
from(this.categoriesApi.createSubcategory(parentCategoryId, payload, {}));

where this.categoriesApi.createSubcategory returns Promise<CategoryEntry> type so from where CateegoryPaging comes?

I think you should not use createSubcategory because it is for creating single category and it is why it returns CategoryEntry. You need probably write new function in js-api to handle multiple categories.

Reply:
updated jsAPI here https://github.com/Alfresco/alfresco-js-api/pull/1549 to make it more clear